### ISSUE TITLE
failing deploy in CI should return an error code

### ISSUE THREAD

**Issue by moldhouse** [Aleph-Alpha/pharia-kernel-sdk-py#35]
if we call `pharia-skill deploy` in ci and the deploy fails, the GH action still shows up as green, which it should not
